index.bs

```diff
@@ -9824,6 +9824,71 @@ Methods</h5>
 		</div>
 </dl>
 
+<h5 id="AudioWorkletProcessor-instantiation">
+The instantiation of AudioWorkletProcessor</h5>
+
+At the end of the {{AudioWorkletNode}} construction,
+A [=struct=] named
+<dfn>processor construction data</dfn>
+will be prepared for the cross-thread transfer. This
+<a spec="infra" lt="struct">struct</a> contains the following
+<a spec="infra" for="struct" lt="item">items</a>:
+
+- <dfn for="processor construction data">name</dfn> which is a {{DOMString}}
+	that is to be looked up in the
+	<a>node name to processor constructor map</a>.
+- <dfn for="processor construction data">node</dfn> which is a reference to
+	the {{AudioWorkletNode}} created.
+- <dfn for="processor construction data">options</dfn> which a serialized
+	{{AudioWorkletNodeOptions}} given to the {{AudioWorkletNode}}'s
+	{{AudioWorkletNode()|constructor}}.
+- <dfn for="processor construction data">port</dfn> which a serialized
+	{{MessagePort}} paired with the {{AudioWorkletNode}}'s
+	{{AudioWorkletNode/port}}.
+
+Upon the arrival of the transferred data on the {{AudioWorkletGlobalScope}},
+the <a>rendering thread</a> will invoke the algorithm below:
+
+	<div id="invoking-processor-constructor" algorithm="invoking processor constructor">
+		1. Let <var>constructionData</var> be the
+			[=processor construction data=] transferred from the
+			[=control thread=].
+
+		1. Let <var>processorName</var> be <var>constructionData</var>'s
+			[=processor construction data/name=].
+
+		1. Let <var>nodeReference</var> be <var>constructionData</var>'s
+			[=processor construction data/node=].
+
+		1. Let <var>serializedPort</var> be <var>constructionData</var>'s
+			[=processor construction data/port=].
+		
+		1. Let <var>serializedOptions</var> be
+			<var>constructionData</var>'s
+			[=processor construction data/options=].
+		
+		1. Let <var>deserializedPort</var> be the result of
+			[$StructuredDeserialize$](<var>serializedPort</var>,
+			the current Realm).
+
+		1. Let <var>deserializedOptions</var> be the result of
+			[$StructuredDeserialize$](<var>serializedOptions</var>,
+			the current Realm).
+		
+		1. Let <var>processorCtor</var> be the result of looking
+			up  <var>processorName</var> on the
+			{{AudioWorkletGlobalScope}}'s
+			<a>node name to processor constructor map</a>.
+		
+		1. Make <dfn><var>nodeReference</var></dfn>,
+			<dfn><var>deserializedPort</var></dfn>, and
+			<var>deserializedOptions</var> available
```

Comment:
This is a new concept of "making things available" for the AWP constructor.

Reply:
Sure. I would like to merge the progress in this PR, and follow-up on this part later.